src/app/day-calendar/day-calendar.component.ts

```diff
@@ -336,4 +350,12 @@ export class DayCalendarComponent implements OnInit, OnChanges, ControlValueAcce
       }
     }
   }
+
+  displayDayCalendar(): boolean {
+    return (this.currentCalendarMode === this.CalendarMode.Day) && (this.monthIsSelect || this.calendarTypeDisplayFirst === 'day')
```

Comment:
please split to variables to make the logic clearer

Reply:
We also better to return here a type of the calendar we want to show. This way we wouldn't  need `displayMonthCalendar` method.